client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -169,19 +170,128 @@ const InstructionsRenderer = ({
+                                aria-labelledby={
+                                    renderableContent.commands[i].keystroke
+                                }
+                            >
+                                <thead>
+                                    <tr>
+                                        <th>Priority</th>
+                                        <th>Assertion Phrase</th>
+                                        <th>Assertion Statement</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    {renderableContent.assertions.map(
+                                        ({
+                                            assertionPhrase,
+                                            assertionStatement,
+                                            priority,
+                                            assertionId
+                                        }) => (
+                                            <tr key={`${i}-${assertionId}`}>
+                                                <td>
+                                                    {convertAssertionPriority(
+                                                        priority
+                                                    )}
+                                                </td>
+                                                <td>{assertionPhrase}</td>
+                                                <td>{assertionStatement}</td>
+                                            </tr>
+                                        )
+                                    )}
+                                </tbody>
+                            </Table>
+                        </>
+                    );
+                } else {
+                    return (
+                        <>
+                            <Heading>{scenarioTitle}</Heading>
+                            <Table
+                                key={`${id}-${i}`}
+                                bordered
+                                responsive
+                                aria-label={`Results for test ${test.title}`}
+                            >
+                                <thead>
+                                    <tr>
+                                        <th>Priority</th>
+                                        <th>Assertion Statement</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    {renderableContent.assertions.map(
+                                        ({
+                                            expectation,
+                                            priority,
+                                            assertionId
+                                        }) => (
+                                            <tr key={`${i}-${assertionId}`}>
```

Comment:
```suggestion
                                        ({
                                            expectation,
                                            priority
                                        }) => (
                                            <tr key={`${i}-${expectation}`}>
```

When looking at the console for plans using the v1 test format version, an error being shown is that there is multiple instances of `{index}-undefined` being set. That's because `assertionId` isn't available [for v1](https://github.com/w3c/aria-at/wiki/Test-Format-V1-Definition#assertion1-through-assertion6) tests but it is [for v2](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2#assertionid).

The suggestion or similar should work